controllers/storagecluster/exporter.go

```diff
@@ -1107,3 +1117,39 @@ func (r *StorageClusterReconciler) deleteMetricsExporterCephClient(namespace str
 
 	return nil
 }
+
+func getRookCephHostNetwork(instance *ocsv1.StorageCluster) bool {
+	if instance.Spec.HostNetwork && instance.Spec.Network != nil {
+		if instance.Spec.Network.AddressRanges != nil && instance.Spec.Network.AddressRanges.Public != nil {
+			return true
+		}
+	}
+	return false
+}
+
+func bindExporterToHostNetworkV2SCC(ctx context.Context, r *StorageClusterReconciler, ns string) error {
+	crb := &rbacv1.ClusterRoleBinding{
```

Comment:
can you please add details why we need to create new clusterRoleBinding is this for metrics exporter to talk to prometheus?